cypress/integration/team-members/profile.js

```diff
@@ -0,0 +1,108 @@
+describe('Team members profile info', () => {
+  it('Should not have permissions when not logged in', () => {
+    cy.visit('http://localhost:3000/team-members/profile/admin_member6@gmail.com')
+
+    cy.contains('Welcome to SalesWell')
+  })
+
+  it('Should not have delete button when logged in as user', () => {
```

Comment:
if you want to combine both tests in one (which is ok), then the name should be 'Should not be able to delete a team member if not admin'